client/src/outside/components/EhrSeedDataList.vue

```diff
@@ -21,7 +21,8 @@
               td {{sv.name}}
               td {{sv.version}}
               td {{sv.description}}
-              td {{ ehrPages(sv) }}
+              td(v-if="shouldBeLimited(sv)", :title="ehrPages(sv)") {{ limitedPages(sv) }}
+              td(v-else) {{ ehrPages(sv) }}
```

Comment:
It'd be cleaner if all the logic was placed in limitedPages() and don't forget to use place content of ehrPages into the hover (title) attribute